.github/workflows/pyinstaller_gui_mac.yaml

```diff
@@ -0,0 +1,25 @@
+name: Compile GUI for Mac
```

Comment:
Let's hold off on deploying workflows to master that haven't been tested. We can chat about the various ways to accomplish this on the side.

Reply:
1. I have tested this on the other [repo](https://github.com/nh916/tkinter-and-eel) that I was originally working on
2. I have the action to run `"on: workflow_dispatch"`, which means that this action only runs when I manually go into GitHub and run it